_includes/integrations/webcomponent-quick-start.md

```diff
@@ -0,0 +1,60 @@
+## TinyMCE Web Component quick start guide
+
+The [Official {{site.productname}} Web Component](https://github.com/tinymce/tinymce-webcomponent) integrates {{site.productname}} into Web Component projects.
+This procedure creates a HTML page containing a {{site.productname}} editor based on our [Basic example]({{site.baseurl}}/demo/basic-example/).
+
+### Procedure
+
+To add a {{site.productname}} editor to a web page using the {{site.productname}} Web Component:
+
+1. Add the `DOCTYPE` element to the target page, such as:
+
+    ```html
+    <!DOCTYPE html>
+    ```
+
+    The `DOCTYPE` element is required for the editor to function correctly.
```

Comment:
Is element the right word for this? All I can see being used to describe it on mdn / the spec is "preamble"

Reply:
It might be close enough. I'd go with "declaration".

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1659/commits/35809236642e6faf2771966c7af9ba7c0d29d940